src/axom/inlet/Table.cpp

```diff
@@ -193,5 +192,47 @@ bool Table::required()
   return (bool)intValue;
 }
 
+ std::shared_ptr<Table> Table::registerVerifier(std::function<bool()> lambda) {
+  SLIC_WARNING_IF(verifier, fmt::format("Verifier for Table {0} already set", 
+                                         m_sidreGroup->getPathName()));
+  verifier = lambda;
+  return shared_from_this();
+ } 
+
+ bool Table::verify() {
+   bool verified = true;
+   // Verify this Table
+   if (verifier && !verifier()) {
+     verified = false;
+     SLIC_WARNING(fmt::format("Table {0} failed verification", m_sidreGroup->getPathName()));
+   }
+   // Verify the child Fields of this Table
+   for (auto field : m_fieldChildren) {
+     if (!field.second->verify()) {
+       verified = false;
+     }
+   }
+   // Verify the child Tables of this Table
+   for (auto table : m_tableChildren) {
+     if (!table.second->verify()) {
+       verified = false;
+     }
+   }
+   
+   return verified;
+ }
+
+bool Table::hasField(const std::string& fieldName) {
+  return m_fieldChildren.find(getFullName(m_name, fieldName)) != m_fieldChildren.end();
+}
+
+bool Table::hasTable(const std::string& tableName) {
+  return m_tableChildren.find(getFullName(m_name, tableName)) != m_tableChildren.end();
+}
+
+std::string Table::getPathName() {
```

Comment:
getPath, Table::getPathName, and getFullName are a bit confusingly named.

'path' doesn't make sense for Inlet as it is a convention of Sidre.  For Inlet, to make things easy on ourselves we should only report the fully qualified variable name without the Sidre prefix.

For example, the user may have an input deck like:

```
foo.bar = 3
```

but if they choose to store their inlet tree under the sidre group "serac/inlet", something that is not exposed to the user on the Serac level, throwing a warning like the following might confuse the user that something else has gone wrong:

```
Warning: Inlet variable "serac/inlet/foo/bar" has an invalid number.
```

Can we simplify this to two name conventions? getFullSidrePath and getName?